taskcluster/darwin-opt-base.tyml

```diff
@@ -103,6 +103,7 @@ payload:
         ln -s $TASKCLUSTER_TASK_DIR/DeepSpeech/ds/native_client/ $TASKCLUSTER_TASK_DIR/DeepSpeech/tf/native_client &&
         cd $TASKCLUSTER_TASK_DIR &&
         $TASKCLUSTER_TASK_DIR/DeepSpeech/tf/tc-brew.sh &&
+        $TASKCLUSTER_TASK_DIR/homebrew/bin/brew reinstall swig &&
```

Comment:
This was done to avoid caching issues during the PR. Final merge should require purging caches everywhere and removing that line.